.github/workflows/update.yml

```diff
@@ -6,13 +6,10 @@ on:
 
 jobs:
   lockfile:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-      - name: Install Nix
-        uses: DeterminateSystems/nix-installer-action@main
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Update flake.lock
```

Comment:
Looks like your automation didn't account for actions with a `name` associated -- this should be attached to the `update-flake-lock` action, not the checkout action ;P